website/docs/error-reference.md

```diff
@@ -1510,6 +1811,15 @@ T.unsafe(self).foo(xs)
 # ---------------------------------------------
 ```
 
+## 7020
+
+Note that method-level generics (i.e., those declared by using `type_parameters`
+inside a `sig`) are a somewhat unstable feature. If you encounter this error and
+believe it to represent a bug in Sorbet, first check the
+[Sorbet bug tracker](https://github.com/sorbet/sorbet/issues) to see if a
+similar-looking error has already been reported. If no such bug exists, feel
+free to open an issue.
+
 ## 7021
 
 The method called declares a block parameter that is not `T.nilable`, but a
```

Comment:
(Noticed in passing) Kind of want to make this "The called method..." so it's more obvious that "called" is not a verb here.